### ISSUE TITLE
correct command layout for using chia-plot

### ISSUE THREAD

**Issue by we-sell-bags** [madMAx43v3r/chia-plotter#101]
Hi,
can we have a dummy example

chia_plot -t /big_store/ -t2 /media/ramdisk/ -r 12 -p........

terminate called after throwing an instance of 'cxxopts::option_requires_argument_exception'
  what():  Option ‘t’ requires an argument

**Comment by fermosan** [comment]
it actually -2 (not -t2).

**Comment by we-sell-bags** [comment]
blind bastard.....;-)
might be better to call them something normal like
-t1 & -t2


thanks

**Comment by madMAx43v3r** [comment]
it's the same as `chia plot ...`

**Comment by we-sell-bags** [comment]
nothing beats taking a bad idea and propagating it......

**Comment by chris5414** [comment]
Hi

I have this 'command not found' , what is wrong with this? 

chia_plot -p 8c6179b13845fb9bd1a00750d6519a9a3404ccae7bb1c721deddfedb2a8f837e912e47275f19dbfcb87ce834a465ef47 -f a58b321b5c8e117d4b2550ace8e217088cd476f050e3d5ba1dc62ae36868d7wd65d79059b882dc0a43c8fbeb11215bf2 -t /media/chris/WD 10Tb Hdd Temp/ -2 /media/chris/WD 1Tb Green/ -d /media/chris/WD 10Tb Hdd Plot/ -n 1 -r 20

**Comment by we-sell-bags** [comment]
you may need teh " full path" for "chia_plot"

like {something here}/build/chia_plot

**Comment by chris5414** [comment]
Still can't get it sorry, I copy the whole thing:

chris@chris-X9AR7-E-J-LN4F:~/chia-plotter/build$ chia_plot -p 8c6179b13845fb9bd1a00750d6519a9a3404ccae7bb1c721deddfedb2a8f837e912e47275f19dbfcb87ce834a465ef47 -f a58b321b5c8e117d4b2550ace8e217088cd476f050e3d5ba1dc62ae36868d7wd65d79059b882dc0a43c8fbeb11215bf2 -t /media/chris/WD 10Tb Hdd Temp/ -2 /media/chris/WD 1Tb Green/ -d /media/chris/WD 10Tb Hdd Plot/ -n 1 -r 20
chia_plot: command not found

**Comment by we-sell-bags** [comment]
don't screw about with ~

go to the directory where "chia_plot" is located
do .
`pwd` and insert that into teh path

**Comment by chris5414** [comment]
Thanks alot, and sorry for being a noob.

Its getting better, now I get passed that stage. This is the new error I got:

Invalid <tmpdir>: /media/chris/WD (needs trailing '/' or '\')

Any idea what is wrong again?

**Comment by we-sell-bags** [comment]
i do it mysellf ALL the time  
you are missing the trailing "/'  
ALSO!!!!!
watch our for devices with spaces in
like:
/home/bob/this is bobs folder

linux uses spaces to separate arguments in commands
you can have a really shitty day
if you are using "del" with such things

you may need 
-t "/media/chris/WD 10Tb Hdd Temp/"
-2 "/media/chris/WD 1Tb Green/"

to tell linux DONT start to reparse at the spaces

WD_1Tb_Green
or
WD-1Tb-Green
is WAY safer for a directory name

**Comment by VertiHydro** [comment]
Yep - you need to write "/media/chris/WD/" not "/media/chris/WD" . Note the slash at the end.

**Comment by chris5414** [comment]
Yes it finally works, thank you guys for your help!!!
